4_advanced_usage/30_technique_creation.txt

```diff
@@ -5,7 +5,94 @@ configuration and system administration needs. You can also create your own
 Techniques, to implement new functionality or configure new services. This
 paragraph will walk you through this process.
 
-==== Prequisites
+==== Use ncf-builder
```

Comment:
I think we must not use ncf-builder in Rudder documentation for anything but the "technical name" of the Technique Editor. It's that last name which is used all around Rudder, and people may be lost if we use anything else. 

Moreover, I would like to have something in the title enforcing that this is the default, recommanded solution, something like:


"""
There is two of configuring new Techniques, either thanks to the web Technique Editor in Rudder or by coding them by hand. 

Using the Technique Editor (code name: ncf-builder [+link to the project] is the easiest way of creating new Technique and is fully integrated with Rudder. On the other hand, it does not allow the same level of complexity and expressivness than coding a Technique by hand. Of course, coding new Technique by hand is a more involved process that need to learn how the Technique description language works. 

We advice to always start to try to create new Technique with the Technique Editor and switch to the hand-coding creation only if you discover specific needs not addressed that way. 

==== Recommanded solution: Technique Editor

...